code/modules/client/preference/preferences_mysql.dm

```diff
@@ -50,7 +50,7 @@
 
 	//Sanitize
 	ooccolor		= sanitize_hexcolor(ooccolor, initial(ooccolor))
-	UI_style		= sanitize_inlist(UI_style, list("White", "Midnight"), initial(UI_style))
+	UI_style		= sanitize_inlist(UI_style, list("White", "Midnight","Plasmafire","Retro","Slimecore","Operative"), initial(UI_style))
```

Comment:
```suggestion
	UI_style		= sanitize_inlist(UI_style, list("White", "Midnight", "Plasmafire", "Retro", "Slimecore", "Operative"), initial(UI_style))
```
We put one space after each operator and `,` to separate values. It makes reading the code a bit easier.